### PR TITLE
refactor: remove unused struct field

### PR DESCRIPTION
**Pull request by james-milligan**
## This PR

- removes unused struct field in filepath sync

### CODE REVIEW

File: pkg/sync/file/filepath_sync.go [hunk 1]
```diff
@@ -17,8 +17,8 @@ import (
 )
 
 type Sync struct {
-	URI          string
-	Source       string
+	URI string
```
Comment (beeme1mr) [inline pkg/sync/file/filepath_sync.go:20]: It looks like this section needs to be formatted.

**Comment by Kavindu-Dodan** [comment]
Can you also check K8s sync - https://github.com/open-feature/flagd/blob/main/pkg/sync/kubernetes/kubernetes_sync.go#L33

File: pkg/sync/file/filepath_sync.go (hunk 1 above)
Comment (james-milligan) [reply pkg/sync/file/filepath_sync.go]: done

**Review by Kavindu-Dodan — APPROVED** [review]

**Review by beeme1mr — APPROVED** [review]

File: pkg/sync/kubernetes/kubernetes_sync.go [hunk 2]
```diff
@@ -31,7 +31,6 @@ type Sync struct {
 	Logger       *logger.Logger
 	ProviderArgs sync.ProviderArgs
 	URI          string
-
 	Source     string
```
Comment (Kavindu-Dodan) [inline pkg/sync/kubernetes/kubernetes_sync.go:34]: `source` is still here

Comment (Kavindu-Dodan) [reply pkg/sync/kubernetes/kubernetes_sync.go]: fixed with latest commit

**Pull request merged**